packages/manager-components/src/components/region/region.component.tsx

```diff
@@ -0,0 +1,25 @@
+import { useTranslation } from 'react-i18next';
+import './translations/region';
+import './translations/datacenter';
+
+export interface RegionProps {
+  mode: string;
+  name: string;
+  micro?: number;
+}
+
+export function Region({ mode = 'region', name, micro }: RegionProps) {
+  const { t: tRegion } = useTranslation('region');
+  const { t: tDatacenter } = useTranslation('datacenter');
```

Comment:
**nitpick:** depending on the mode, you only need to load one among these.
```suggestion
  const { t } = useTranslation(mode === 'region' ? 'region' : 'datacenter');
```

Reply:
Yes you are right or else it loads both. I will fix it.